BamaCharanChhandogi/GitFinder
# added a "Star us on GitHub" button: (This PR resolves Issue #6)

### Description
This PR adds a "Star us on GitHub" button to the `Hero` component to allow users to star the repository and drive engagement quickly.

---

### Changes
- A button was added at the top right corner of the `Hero` component.
- The button includes the Star logo and text, "Star us on GitHub".
- The button links to the GitFinder repository on GitHub.

---

### How to Test
1. Run the application.
2. Verify the "Star us on GitHub" button is visible in the top right corner.
3. Click the button to ensure it redirects to the GitFinder GitHub repository.

![Screenshot 2024-10-02 031915](https://github.com/user-attachments/assets/2831bfac-b19b-4b8a-807e-38b210ca6cfd)
